language/tools/move-package/src/source_package/parsed_manifest.rs

```diff
@@ -38,7 +39,21 @@ pub struct Dependency {
     pub local: PathBuf,
     pub subst: Option<Substitution>,
     pub version: Option<Version>,
-    pub digest: Option<Vec<u8>>,
+    pub digest: Option<PackageDigest>,
+    pub git_info: Option<GitInfo>,
+}
+
+#[derive(Debug, Clone, Eq, PartialEq)]
+pub struct GitInfo {
+    /// The git clone url to download from
+    pub git_url: Symbol,
+    /// The git reference -- a commit SHA or a branch name, e.g., 'main'
+    pub git_ref: Symbol,
+    /// The path under this repo where the move package can be found -- e.g.,
+    /// 'language/move-stdlib`
+    pub subdir: PathBuf,
```

Comment:
Is the subdir something that the use has to specify or do you try to be smart like cargo and search the repo for the package?

Reply:
The subdir does not need to be specified if the git repo is a Move package, e.g., https://github.com/tzakian/ExamplePkg. You only need the `subdir` to be supplied if the package is nested within that repo (we don't try to do anything smart, and I'd like to try and avoid that at least for now).